src/tests/e2e/pages/receipt.page.js

```diff
@@ -422,13 +418,11 @@ class ReceiptPage {
         .unix(transfer.chainsferToSender.txTimestamp)
         .format('MMM Do YYYY, HH:mm:ss')
       expect(cancelTime).toEqual(`Cancelled on ${expectCancelTime}`)
-      const cancelExplorerPage = await getNewPopupPage(browser, async () => {
-        await this.dispatchActions('openCancelExplorer')
-      })
-      expect(cancelExplorerPage.url()).toEqual(
+
+      const cancelLink = (await this.getReceiptFormInfo('cancelOn')).cancelOnExplorerLink
```

Comment:
Shouldn't we make sure the button actually opens the explorer page?

Reply:
frequent access to etherscan causing use to be blocked and navigation timeout